### ISSUE TITLE
[MRRESOURCES-125] Unexpected warnings "Invalid project model for artifact [sisu-guice:org.sonatype.sisu:3.2.3]. It will be ignored by the remote resources Mojo."

### ISSUE THREAD

**Issue by jira-importer** [apache/maven-remote-resources-plugin#226]
**[Guillaume Nodet](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=gnt)** opened **[MRRESOURCES-125](https://issues.apache.org/jira/browse/MRRESOURCES-125?redirect=false)** and commented

I have a bunch of warnings when building m-plugin-tools, during the m-remote-resources-p execution.  They look like

```
[WARN] Invalid project model for artifact [sisu-guice:org.sonatype.sisu:3.2.3]. It will be ignored by the remote resources Mojo.
```

but are related to errors  like

```
[ERROR] Failed to determine Java version for profile jdk7-plugin-fix-version @ org.apache.commons:commons-parent:39, /Users/gnodet/.m2/repository/org/apache/commons/commons-parent/39/commons-parent-39.pom, line 1343, column 14
```

The pom looks like:

```
<activation>
  <jdk>[1.7,)</jdk>
</activation>
```

Also the warning uses `artifactId:groupId:version` instead of the more used `groupId:artifactId:version`

---

**Affects:** 3.0.0

**Issue Links:**
- [MRRESOURCES-121](https://issues.apache.org/jira/browse/MRRESOURCES-121) version 3.0.0 fail to include some dependencies
 (_**"is fixed by"**_)
- [MDEP-630](https://issues.apache.org/jira/browse/MDEP-630) error dependency:list (caused by postgresql dependency)

- [MPIR-429](https://issues.apache.org/jira/browse/MPIR-429) avoid warning on "Unknown packaging: bundle"

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

same cause as MPIR-429 and MDEP-630: same solution to be applied :)

**Comment by jira-importer** [comment]
**[Hervé Boutemy](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=hboutemy)** commented

Olivier already fixed it in MRRESOURCES-121 , just requires 3.1.0 to be released
